packages/x-date-pickers/src/tests/describeGregorianAdapter/testLocalization.ts

```diff
@@ -12,9 +12,11 @@ export const testLocalization: DescribeGregorianAdapterTestSuite = ({ adapter })
   });
 
   it('Method: getFormatHelperText', () => {
-    expect(adapter.getFormatHelperText(adapter.formats.keyboardDate)).to.equal('mm/dd/yyyy');
+    expect(adapter.getFormatHelperText(adapter.formats.keyboardDate)).to.equal(
+      adapter.lib === 'luxon' ? 'm/d/yyyy' : 'mm/dd/yyyy',
+    );
     expect(adapter.getFormatHelperText(adapter.formats.keyboardDateTime12h)).to.equal(
-      'mm/dd/yyyy hh:mm (a|p)m',
+      adapter.lib === 'luxon' ? 'm/d/yyyy hh:mm (a|p)m' : 'mm/dd/yyyy hh:mm (a|p)m',
```

Comment:
This method was not working on Luxon on `date-io` so it's not a behavior change to change the test, it was just never used with Luxon :+1: 

By the way, the method is deprecated and we never use it